kie-soup-dataset/kie-soup-dataset-sql/src/test/java/org/dashbuilder/dataprovider/sql/SQLColumnsTypeTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dashbuilder.dataprovider.sql;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.dashbuilder.dataset.DataColumn;
+import org.dashbuilder.dataset.DataSet;
+import org.dashbuilder.dataset.def.SQLDataSetDef;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SQLColumnsTypeTest extends SQLDataSetTestBase {
+    
+
+    String CLOB_TABLE = "CLOB_TABLE";
+    String CLOB_COLUMN = "CLOB_CL";
+    String CLOB_VAL = "TEST_CLOB";
+
+    @Before
+    public void prepareForClobTest() throws SQLException {
+        String TABLE_SQL = "CREATE TABLE "+ CLOB_TABLE +" ("
+                + "ID INTEGER PRIMARY KEY,"
+                + CLOB_COLUMN + " CLOB);";
+        String INSERT = "INSERT INTO " + CLOB_TABLE + " VALUES(1, '"+ CLOB_VAL + "')";
+        Statement stmt = conn.createStatement();
+        stmt.executeUpdate(TABLE_SQL);
+        stmt.executeUpdate(INSERT);
+    }
+    
+    @Override
+    public void tearDown() throws Exception {
+        String DELETE = "DROP TABLE " + CLOB_TABLE;
+        Statement stmt = conn.createStatement();
+        stmt.executeUpdate(DELETE);
+        super.tearDown();
+    }
+    
+    @Test
+    public void clobColumnTest() throws Exception {
+        
+        SQLDataSetDef def = new SQLDataSetDef();
+        def.setDbTable(CLOB_TABLE);
+        
+        def.setDataSource("jdbc:h2:mem:test");
```

Comment:
Is this going to work with different datasources?

Reply:
@domhanak It should work with multiple datasources, but this is the unit test for CLOB. Do you want to include a CLOB test in the kie-soup-datasource-sql-tests ?

Reply:
is that feasible? Could be nice to cover it there too.

Reply:
I just included it in the tests ran by kie-soup-dataset-sql-tests - so it should run for all databases - I found an issue with MySQL: in MySQL CLOB isn't a direct type, instead it is used behind some text types - hence the create table command had to be adjusted. For most databases it will run correctly (noticed it works with Oracle and DB2, for example) - but if it fails let me know and I will adjust the SQL - I think MySQL/MariaDB were the only exception :-) 

Let me know if you see any issue.